src/impls/$distinct/DOCME.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "transform",
+  "signatures": [[{ "name": "selector" }], []]
```

Comment:
You don't need to do this -- the documentation engine will infer it in all but the weirdest cases. I'm guessing it just did something weird because `maxArgs` was off.

Reply:
fixed in 64f50880549afd41939f95942a41006ffd65ce5d